app/components/Instructions/Instructions.js

```diff
@@ -2,57 +2,55 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 export default function Instructions({ state, }) {
-  function buildInstructionArray(appState) {
+  function createInstructionArray(appState) {
```

Comment:
"Array" is an implementation detail. The caller shouldn't know it.

Reply:
done